composer.json

```diff
@@ -3,7 +3,8 @@
     "require": {
         "silex/silex": "^2.2",
         "twig/twig": "^2.4",
-        "doctrine/orm": "^2.5"
+        "doctrine/orm": "^2.5",
+        "mockery/mockery": "^0.9.9"
```

Comment:
mockery is in require, it would be better to move it to the require-dev block below (so it doesn't get installed in production environments)